core/services/fluxmonitorv2/contract_submitter.go

```diff
@@ -58,7 +60,7 @@ func (c *FluxAggregatorContractSubmitter) Submit(roundID *big.Int, submission *b
 	}
 
 	return errors.Wrap(
-		c.orm.CreateEthTransaction(fromAddress, c.Address(), payload, c.gasLimit, c.maxUnconfirmedTransactions),
+		c.orm.CreateEthTransaction(db, fromAddress, c.Address(), payload, c.gasLimit, c.maxUnconfirmedTransactions),
```

Comment:
I feel like this should be transactional with marking the log consumed

Reply:
it is https://github.com/smartcontractkit/chainlink/blob/8bc2e76471b608eda39f7ad330b33df4197460fa/core/services/fluxmonitorv2/flux_monitor.go#L687